.github/install_open_csp.sh

```diff
@@ -275,6 +275,7 @@ function do_composer()
     $COMPOSER config repositories.38 composer https://gitlab.wikibase.nl/api/v4/group/38/-/packages/composer/ || exit_with_message
     $COMPOSER update --no-dev || exit_with_message
     $COMPOSER update --no-dev || exit_with_message
+    chmod -v +x extensions/Scribunto/includes/engines/LuaStandalone/binaries/lua5_1_5_linux_64_generic/lua
```

Comment:
Waarom `-v`?

Reply:
Omdat we een log genereren hiermee. De gebruiker kan alles zien wat er gebeurt. Niet noodzakelijk, maar lijkt me wel logisch.